qmake/compiler.pri

```diff
@@ -136,8 +136,10 @@ win32-msvc* {
 	# For our non-64 x86 builds, our binaries should not contain any
 	# SSE2 code, so override the default by using -arch:SSE.
 	equals(MUMBLE_ARCH, x86) {
-		QMAKE_CFLAGS_RELEASE *= -arch:SSE
-		QMAKE_CXXFLAGS_RELEASE *= -arch:SSE
+		!use-x86-sse {
```

Comment:
This seems to be a new build-option. Please document it in the `INSTALL` file at the project's root